server/gitlab/pipeline.go

```diff
@@ -0,0 +1,22 @@
+package gitlab
+
+// PipelineInfo is the struct describing the status of
+// a running pipeline.
+type PipelineInfo struct {
+	PipelineID int
+	Status     string
+	Ref        string
+	WebURL     string
+	SHA        string
+	User       string
+}
+
+// PipelineInfo is the struct describing the status of
+// a running Job which is part of a pieline
```

Comment:
@spirosoik - `pieline`->`pipeline`; Also, do we need to include the `PipelineID` field like for subscription?

Reply:
Thanks for this @catalintomai. No needed I have added a subscription to get status updates but let's do it in a different PR to keep it small.